examples/svelte-kitchen-sink/svelte.config.js

```diff
@@ -0,0 +1,5 @@
+const sveltePreprocess = require('svelte-preprocess');
+
+module.exports = {
+  preprocess: sveltePreprocess(),
```

Comment:
what's the purpose of this `preprocess` vs. the call in `main.js/svelteOptions`?

Reply:
`svelte.config.js` is an unofficial configuration file used by some tools to centralize the configuration of the svelte compiler.

It's not used by svelte itself, but by the vscode extension, or here by `svelte-jester`.

This file is undocumented/unofficial, so it didn't try to integrate it into storybook. it's just here in the svelte-kitchen-sink to make svelte-jester work.